### PR TITLE
build on s390x

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
glog 0.5.0 on 390x, dependency for arrow-cpp 14.0.1

### CODE REVIEW

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,7 +15,9 @@ source:
     - 0004-Ignore-unmangled-cxx11-symbols.patch
 
 build:
-  number: 0
+  number: 1
+  missing_dso_whitelist:  # [s390x]
+    - '$RPATH/ld64.so.1'  # [s390x]
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:20]: I would pin `gflags` to `2.2`

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]
The linter warnings and errors are all easy to fix.

**Review by cbouss — COMMENTED** [review]
Can you add gtest as a dependency?
https://github.com/google/glog/blob/v0.5.0/CMakeLists.txt#L71

Also, can you correct the about section?

Thank you

File: recipe/meta.yaml [hunk 2]
```diff
@@ -32,7 +32,8 @@ requirements:
     - patch       # [not win]
     - m2-patch    # [win]
   host:
-    - gflags
+    - gflags 2.2
+    - gtest
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]: You'll need to add it to `ignore_run_exports` unless you want ti to be in the runtime dependencies. @cbouss why did you request gtest to be added?

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml]: My comment is still unaddressed. As far as I see, we don't run the tests so gtest is not needed.

Comment (cbouss) [reply recipe/meta.yaml]: oh right, it's only used if `BUILD_TESTING=ON`
In that case @M-Waszkiewicz-Anaconda can you remove gtest? But then bld.bat should set testing off:
https://github.com/AnacondaRecipes/glog-feedstock/blob/PKG-3988/recipe/bld.bat#L8

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**